tests/chainerx_tests/unit_tests/routines_tests/test_linalg.py

```diff
@@ -227,3 +227,101 @@ def forward_xp(self, inputs, xp):
         a, b = inputs
         out = xp.linalg.solve(a, b)
         return out,
+
+
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize(*(
+    chainer.testing.product({
+        'shape': [(1, 1), (3, 3), (6, 6)],
+        'in_dtypes': ['float32', 'float64']
+    })
+))
+class TestCholesky(NumpyLinalgOpTest):
+
+    def generate_inputs(self):
+        a = numpy.random.random(self.shape).astype(self.in_dtypes)
+        # Make random square matrix a symmetric positive definite one
+        a = numpy.array(a.T.dot(a)) + 1e-6*numpy.eye(*self.shape)
```

Comment:
A nitpick on the style.
```suggestion
        a = numpy.array(a.T.dot(a)) + 1e-6 * numpy.eye(*self.shape)
```